docs/stock/compare.md

```diff
@@ -0,0 +1,12 @@
+Compare
+================
+
+The compare tool is used to compare the values to the first value in a visible range.
```

Comment:
I would change it to:
`The compare tool is used to compare each points' value to the first value in a visible range.`
It's not exactly clear what we are comparing :)

Reply:
PS: We also have [`compareStart`](https://api.highcharts.com/highstock/series.line.compareStart) option :)